gstd/src/macros/debug.rs

```diff
@@ -69,14 +69,15 @@ macro_rules! debug {
 #[macro_export]
 macro_rules! dbg {
     () => {
-        $crate::debug!("[{}:{}]", $crate::prelude::file!(), $crate::prelude::line!())
+        $crate::debug!("[{}:{}:{}]", $crate::prelude::file!(), $crate::prelude::line!(), $crate::prelude::column!())
```

Comment:
isn't it excess info?

Reply:
no, 1.76 added this https://github.com/rust-lang/rust/pull/114962

Reply:
> no, 1.76 added this [rust-lang/rust#114962](https://github.com/rust-lang/rust/pull/114962)

I mean for our case of calling it in gear programs

Reply:
I can revert that but mb column is useful to jump to source file?

Reply:
not a big deal